crates/sui-indexer/migrations_v2/2023-08-19-044023_objects/up.sql

```diff
@@ -57,8 +57,11 @@ CREATE TABLE objects_history (
     df_object_id                bytea,
     CONSTRAINT objects_history_pk PRIMARY KEY (checkpoint_sequence_number, object_id, object_version)
 ) PARTITION BY RANGE (checkpoint_sequence_number);
+CREATE INDEX objects_history_owner ON objects_history (owner_type, owner_id) WHERE owner_type BETWEEN 1 AND 2 AND owner_id IS NOT NULL;
+CREATE INDEX objects_history_coin ON objects_history (owner_id, coin_type) WHERE coin_type IS NOT NULL AND owner_type = 1;
+CREATE INDEX objects_history_type ON objects_history (object_type);
+-- init with first partition of the history table
```

Comment:
should these also be prefixed with checkpoint_sequence_number? For objects_history_owner, I would assume that we want owner_id before owner_type - like objects_history, unless it doesn't matter?

And I think we're also missing the idx_objects_history (checkpoint_sequence_number, object_id, object_version DESC)

Reply:
```
idx_objects_history (checkpoint_sequence_number, object_id, object_version DESC)
```
is the primary key (except that primary key does not take DESC syntax)

Reply:
great point about the cp_seq_number, editing it now